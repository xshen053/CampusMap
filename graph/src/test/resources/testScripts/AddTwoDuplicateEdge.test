# add two same edges, see if it will print one edge or two

# Create a graph
CreateGraph ATDW_graph

# Add two nodes
AddNode ATDW_graph a
AddNode ATDW_graph b

# Add one edge
AddEdge ATDW_graph a b e1
AddEdge ATDW_graph a b e1

# Print the nodes in the graph and the outgoing edges from a
ListNodes ATDW_graph
ListChildren ATDW_graph a