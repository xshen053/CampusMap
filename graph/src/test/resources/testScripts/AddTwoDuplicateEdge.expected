# add two same edges, see if it will print one edge or two

# Create a graph
created graph ATDW_graph

# Add two nodes
added node a to ATDW_graph
added node b to ATDW_graph

# Add one edge
added edge e1 from a to b in ATDW_graph

# Print the nodes in the graph and the outgoing edges from a
ATDW_graph contains: a b
the children of a in ATDW_graph are: b(e1)